cpp/src/io/json/experimental/read_json.cpp

```diff
@@ -161,6 +161,13 @@ table_with_metadata read_json(host_span<std::unique_ptr<datasource>> sources,
                  "specifying a byte range is supported only for json lines");
```

Comment:
Can we standardize this error message with the other one added in this PR? Generally, we should be capitalizing JSON and decide whether to use `lines` or `Lines` consistently across the codebase.
```suggestion
                 "specifying a byte range is supported only for JSON Lines");
```